setup.py

```diff
@@ -20,6 +20,7 @@
   include_package_data = True,
   install_requires = [
     'click',
+    'jsonschema',
```

Comment:
@lukasheinrich if possible can you add a known lower bound for `jsonschema` here?

Reply:
`pip freeze` gives me `jsonschema==3.2.0` but idk whether lower bounds work too

Reply:
i'll mark this as a new issue to figure out a sane setup.py